src/desktop/apps/rss/templates/news.jade

```diff
@@ -20,7 +20,7 @@ rss(version='2.0', xmlns:atom='http://www.w3.org/2005/Atom' xmlns:content='http:
           - media = article.get('media')
           enclosure(url=media.url length=0 type="video/mp4")
         else if article.get('layout') === 'news'
-          enclosure(url="#{sd.APP_URL}/images/og_image.jpg" length=0 type="image/jpeg")
+          meta(property="og:image" content="#{sd.APP_URL}/images/og_image.jpg")
```

Comment:
Hmm I wonder if this will affect Flipboard -- I remember using `enclosure` was one of the tags we needed (https://about.flipboard.com/rss-spec/). Would keeping both cause issues?

Reply:
Yeah I think that should be fine, are there any other places that consume the RSS feed? Will add Flipboard to our distro docs, assuming this is used internally by the editorial team?

Reply:
Yep, Flipboard is the latest distribution that editorial directly manages through a publisher account. Feel free to merge after we add back `enclosure`!